README.md

```diff
@@ -3,3 +3,37 @@
 This tool helps migrating from one instance to another.
 
 Currently supported are Mariadb and Redis.
+
+## Migrate MariaDB
+
+The migration can be triggered via `cf push`.
+
+To migrate create the following migrate.yaml:
+
+```yaml
+applications:
+  - name: migrate-mariadb
+    memory: 64M
+    instances: 1
+    services:
+      - mariadb-source
```

Comment:
Just for my understanding... both these services will be in the same `Namespace`?